code/modules/awaymissions/mission_code/ruins/telecomns.dm

```diff
@@ -17,6 +17,36 @@ GLOBAL_LIST_EMPTY(telecomms_trap_tank)
 	GLOB.telecomms_bots -= src
 	return ..()
 
+/mob/living/simple_animal/bot/secbot/buzzsky/telecomms/doomba
+	name = "A FUCKING DOOMBA"
+	desc = "IT'S GOT A BOMB RUN!"
+	var/obj/structure/reagent_dispensers/fueltank/internal_tank
+	var/obj/structure/marker_beacon/dock_marker/collision/decorative_eye
+
+/mob/living/simple_animal/bot/secbot/buzzsky/telecomms/doomba/Initialize(mapload)
+	. = ..()
+	internal_tank = new /obj/structure/reagent_dispensers/fueltank(src)
+	internal_tank.forceMove(src)
+	decorative_eye = new /obj/structure/marker_beacon/dock_marker/collision(src)
+	decorative_eye.forceMove(src)
```

Comment:
Also shouldn't be needed

Reply:
yeah it won't, I was gaslighting myself by not making them click opaque originally which made them appear on right click